src/layouts/MainLayout.tsx

```diff
@@ -34,7 +34,7 @@ export const MainLayout: FunctionComponent = ({ children }) => {
               <Menu />
             </IconButton>
 
-            <img alt="Division LOGO" src={Env.LOGO_URL} height="45" />
+            <img alt="Division LOGO" src={Env.ASSETS_HOST + "/logos/tag_white.png"} height="45" />
```

Comment:
Seria legal também deixar essa concatenação em template literals [assim como na linha 28](src/routes/PublicRoutes.tsx#L28) para manter um padrão no código.

Reply:
> It would be nice replacing it with template literals [like line 28 of PublicRoutes](https://github.com/ivao-brasil/admin.booking.br.ivao.aero/blob/7abba6b1103b35ab739d41b4e3d836aaf7456e15/src/routes/PublicRoutes.tsx#L28) to maintain the code standardized.

yeah, I tried to use template literals for line 37, but for some reason it was not accepted by React as a interpolation string, unfortunately my knowledge on React is very limited, so I don't know how to solve this line out. If anyone knows how to transform this into a template literal string, feel free to commit a refactoring.

Thank you for the feedback @jvbraganca :)

Reply:
```suggestion
            <img alt="Division LOGO" src={`${Env.ASSETS_HOST}/logos/tag_white.png`} height="45" />
```

Does this not work (or generates a syntax error)?